src/less/toolkit.less

```diff
@@ -4,6 +4,6 @@
 
 @import "_variables";
 
-@import "_button";
-@import "_flag";
-@import "_tile";
+@import "../components/button/code/less/_button";
+@import "../components/button/code/less/_flag";
```

Comment:
You're importing the flag less from the button component? Same for the tile.

Could you simplify this by creating a helper function that'll return you the full path provided that you tell it what component you want the path for? (I hope that makes sense)